github/pulls.go

```diff
@@ -134,6 +135,29 @@ func (s *PullRequestsService) Get(owner string, repo string, number int) (*PullR
 	return pull, resp, err
 }
 
+// GetRaw gets raw (diff or patch) format of a pull request.
+func (s *PullRequestsService) GetRaw(owner string, repo string, number int, opt GetRawOptions) ([]byte, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/pulls/%d", owner, repo, number)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return []byte{}, nil, err
```

Comment:
No need to allocate an empty byte slice when returning an error here, just return a `nil` byte slice.

```Go
return  nil, nil, err
```